src/components/MoveModal.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { useI18n } from 'cozy-ui/react/I18n'
+import useInstanceSettings from 'hooks/useInstanceSettings'
+import { withClient } from 'cozy-client'
+
+import { Button } from 'cozy-ui/transpiled/react/Button'
+import { IllustrationDialog } from 'cozy-ui/transpiled/react/CozyDialogs'
+
+const importImage = require('assets/images/moved-cozy.svg')
+
+export const MoveModal = ({ client }) => {
+  const { t } = useI18n()
+  const { data: instanceSettings } = useInstanceSettings(client)
+
+  const movedFrom = instanceSettings['moved_from']
+  const [hasBeenClosed, setClosed] = useState(false)
+  const displayModal = movedFrom && !hasBeenClosed
+
+  const closeModal = () => {
+    setClosed(true)
+    client.getStackClient().fetchJSON('DELETE', '/settings/instance/moved_from')
```

Comment:
This is not awaited, if the DELETE fails, woulnd't it be necessary to 

```
try {
  await client.getStackClient().fetchJSON('DELETE', '/settings/instance/moved_from')
} catch {
  setClosed(false)
}
```

Reply:
Well, it's a personal choice, but I think that if the user closes the modal, it is better to let browse their Cozy instead of forcing them to be stuck on this modal.

Reply:
I am not aware of the design / UX choices from reading the code, can you add a description to the PR ?